### ISSUE TITLE
Start-PPDMprotection not working as expected

### ISSUE THREAD

**Issue by XtremeBaumer** [bottkars/PPDM-pwsh#5]
Hi,

I have identified 2 issues with this function. 

1. Parameter $PolicyObject does not set policy id correctly (protection_policies.psm1). 

```      
'byPolicyObject' {
        $StageID = ($PolicyObject.stages | Where-Object type -eq PROTECTION).id
        $BackupType = ($PolicyObject.stages | Where-Object type -eq PROTECTION).operations.backupType
        $PolicyID = $Policy.id
      }
```
should be:
```
'byPolicyObject' {
        $StageID = ($PolicyObject.stages | Where-Object type -eq PROTECTION).id
        $BackupType = ($PolicyObject.stages | Where-Object type -eq PROTECTION).operations.backupType
        $PolicyID = $PolicyObject.id
      }
```

2. The response is empty, even though the policy is being restarted.
```
[
  "Tue, 18 Oct 2022 11:01:56 GMT",
  null
]
```

This is fixed in the same function. Remove `.Date`:

```
  end {  
    if (!$noop.IsPresent) {

      switch ($PsCmdlet.ParameterSetName) {
        default {
          write-output $response #was write-output $response.Date
        } 
      }   
    }
  }
```
Perhaps it would be even better to use `write-output $response.Results` instead of returning the whole response

**Comment by bottkars** [comment]
Thanks for bringing that up.

Point 1 is fixed in 19.12.1
I don't get Point 2:

The POST response does not return a result, only response headers:
![image](https://user-images.githubusercontent.com/8255007/205895047-e1348b25-fbdf-4702-b62e-6b816b4be868.png)
Thus , the Header Response will pe scoped to response, and decided to use  .date as the output.
If Unsuceesfull, an error will be thrown

**Comment by XtremeBaumer** [comment]
Actually I am getting a different response, which also matches what Dell lists in the [documentation](https://developer.dell.com/apis/4378/versions/19.12.0/docs/tasks/manual-backup.md). 

Can't upload a picture, but the result looks like thi:

```
Key   : Access-Control-Allow-Methods
Value : {PUT, POST, GET, OPTIONS, DELETE}

Key   : Access-Control-Max-Age
Value : {3600}

Key   : Access-Control-Allow-Headers
Value : {x-requested-with}

Key   : X-Frame-Options
Value : {SAMEORIGIN}

Key   : X-XSS-Protection
Value : {1; mode=block}

Key   : X-Content-Type-Options
Value : {nosniff}

Key   : Content-Security-Policy
Value : {script-src 'self'; object-src 'self'; style-src 'self' 'unsafe-inline'; frame-ancestors 'self'}

Key   : Date
Value : {Tue, 06 Dec 2022 11:57:36 GMT}

Key   : Strict-Transport-Security
Value : {max-age=31536000 ; includeSubDomains}

Key   : Keep-Alive
Value : {timeout=60}

Key   : Referrer-Policy
Value : {no-referrer}

Key   : x-dell-log-trace-id
Value : {TRACE_ID:98da8750a9c3e7f1}

Key   : Transfer-Encoding
Value : {chunked}

Key   : Content-Type
Value : {application/json}

results : {@{stageId=1728aff2-7149-2933-c26e-fa31ac6454ef; status=202; activityId=7c6f5852-835a-49b6-9737-a8682276eb3c}}
```

And the `results : {@{stageId=1728aff2-7149-2933-c26e-fa31ac6454ef; status=202; activityId=7c6f5852-835a-49b6-9737-a8682276eb3c}}` is of interest if you want to query for exactly these manually started jobs. 

To get this output, I removed .date and am using PPDM Version: 19.12.0-19
